benchopt/dataset_util.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+from scipy import sparse
+from bz2 import BZ2Decompressor
+from os.path import join as pjoin
+
+from download import download
```

Comment:
Bad import

Reply:
I put it below since it's not a standard library, how should I do ?

Reply:
No I though it was not the right name. I did not know `download`. I would group it with the one above (`numpy` is not a stdlib import either) but this is nitpicking.